src/stylesheets/home..css

```diff
@@ -0,0 +1,22 @@
+.container {
```

Comment:
file name has a typo, should be `home.css` instead of `home..css` (two dots)

Reply:
Done!